app/editor/src/features/admin/work-orders/WorkOrderForm.tsx

```diff
@@ -75,7 +75,7 @@ export const WorkOrderForm: React.FC = () => {
         iconType="back"
         label="Back to WorkOrders"
         className="back-button"
-        onClick={() => navigate('/admin/work/orders')}
+        onClick={() => navigate('/work/orders')}
```

Comment:
Work orders are visible to editors now, but this particular form will remain in the `/admin` code path because that is what it really is for.  Eventually we'll make a form for editors so that the info is presented in a useful way.